js/static_dependencies/qs.js

```diff
@@ -0,0 +1,737 @@
+(function(f){if(typeof exports==="object"&&typeof module!=="undefined"){module.exports=f()}else if(typeof define==="function"&&define.amd){define([],f)}else{var g;if(typeof window!=="undefined"){g=window}else if(typeof global!=="undefined"){g=global}else if(typeof self!=="undefined"){g=self}else{g=this}g.Qs = f()}})(function(){var define,module,exports;return (function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
+'use strict';
+
+var replace = String.prototype.replace;
+var percentTwenties = /%20/g;
+
+module.exports = {
+    'default': 'RFC3986',
+    formatters: {
+        RFC1738: function (value) {
+            return replace.call(value, percentTwenties, '+');
+        },
+        RFC3986: function (value) {
+            return value;
+        }
+    },
+    RFC1738: 'RFC1738',
+    RFC3986: 'RFC3986'
+};
+
+},{}],2:[function(require,module,exports){
+'use strict';
+
+var stringify = require('./stringify');
```

Comment:
how about these? ;)

Reply:
https://travis-ci.org/ccxt/ccxt/builds/510365957#L1489

Reply:
should be solved

Reply:
if you want to mirror what i did copy this file https://github.com/ljharb/qs/tree/master/dist

and all the files in this dir https://github.com/ljharb/qs/tree/master/lib into a dir called qs/ under static dependencies

Reply:
Yeah, however, there's more to it. The `dist/qs.js` is the *"browserified"* version of the `lib` folder, in other words, it's all the same files glued together to run in a browser (because browsers cannot `require()` files from a filesystem). However, ccxt has it's own browserified version, here:

- https://github.com/ccxt/ccxt/blob/master/package.json#L38
- https://github.com/ccxt/ccxt/blob/master/dist/ccxt.browser.js
- https://github.com/ccxt/ccxt/blob/master/ccxt.browser.js

Upon `npm run build` a browserified version of ccxt is generated, which includes all the files `required()` by ccxt. Upon the build it is updated automatically in [unpkg CDN](https://unpkg.com/) here: https://unpkg.com/ccxt

So, if we simply require `js/static_dependencies/qs/index.js`, all of the other files from static_dependendencies/qs/ will be required as well, and upon ccxt-browserify step, all of them will be included into ccxt. That means that we don't need the browserified version of qs (`dist/qs.js`), so we can safely remove that file.